api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -152,14 +152,12 @@ private DbUser initializeUserIfNeeded() {
       // If the user is already registered, their profile will get updated.
       fireCloudService.registerUser(dbUser.getGivenName(), dbUser.getFamilyName());
 
-      // by approving the latest AOU Terms of Service, the user also approves the latest Terra TOS
-      try {
-        userService.validateTermsOfService(dbUser);
+      // getUserTermsOfServiceStatus and acceptTos will take care of scenario where user has not
```

Comment:
the old comment has important context that we should retain here.

This new comment is good too, but it has some grammar issues that we should fix before merging.